test/validator/astnode-cp/ASTNodeCP.test.ts

```diff
@@ -133,30 +133,30 @@ describe('ASTNodeCP', () => {
 			context('for property reassignment.', () => {
 				it('allows assignment directly on objects.', () => {
 					const goal: AST.ASTNodeGoal = AST.ASTNodeGoal.fromSource(`
-						List.<int>([42]).[0]                 = 42;
-						Dict.<int>([i= 42]).[@i]             = 42;
-						Set.<int>([42]).[43]                 = false;
-						Map.<bool, int>([[true, 42]]).[true] = 42;
+						List.<int>((42,)).[0]                 = 42;
+						Dict.<int>((i= 42)).[@i]              = 42;
+						Set.<int>((42,)).[43]                 = false;
+						Map.<bool, int>(((true, 42),)).[true] = 42;
 					`);
 					goal.varCheck();
 					return goal.typeCheck(); // assert does not throw
 				});
 				it('throws when property assignee type is not supertype.', () => {
 					[
 						`
-							let l: mut int[] = List.<int>([42]);
+							let l: mut [int] = List.<int>((42,));
```

Comment:
```suggestion
							let l: mut [int] = [42];
```
also use literal syntax for the dict, set, and map below